x/ccv/provider/keeper/power_shaping.go

```diff
@@ -88,7 +89,6 @@ func (k Keeper) UpdateMinimumPowerInTopN(ctx sdk.Context, consumerId string, old
 }
 
 // CapValidatorSet caps the provided `validators` if chain with `consumerId` is an Opt In chain with a validator-set cap.
-// If cap is `k`, `CapValidatorSet` returns the first `k` validators from `validators` with the highest power.
```

Comment:
I would keep the comment but just remove `with the highest power`.

Reply:
It's returned now without  `with the highest power`.